Failing thermo test?

<!--Thanks for creating this bug report! Please try to provide as much information as possible so we can recreate the bug. This template is provided as a guide, which you can adjust based on your issue. Also, try searching GitHub issues to see if someone has previously posted a similar bug. -->

### Bug Description
I'm getting failures for a thermo test on my computer for some reason, even though it passes on Travis.
```
======================================================================
FAIL: test_symmetry_contribution_radicals (rmgpy.data.thermoTest.TestThermoDatabase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/mjliu/Code/RMG-Py/rmgpy/data/thermoTest.py", line 169, in test_symmetry_contribution_radicals
    self.assertAlmostEqual(thermo_data_lib.get_entropy(298.), thermo_data_ga.get_entropy(298.), 0)
AssertionError: 194.78593339980998 != 194.00958606236847 within 0 places (0.7763473374415071 difference)

----------------------------------------------------------------------
```

I can't figure out exactly why, but confirmed the following behavior:
- Fails on 2.4.1, library value from Lai_Hexylbenzene
- Fails on master, library value from Lai_Hexylbenzene
- Passes if database reverted to before Lai_Hexylbenzene libraries were merged, library value from thermo_DFT_CCSDTF12_BAC

In all cases, the value of the group additivity estimate was identical. I just pushed a branch to figure out what results Travis gets and why it passes without any problems.

### How To Reproduce
Hopefully this is reproducible, and not local to my install/computer. Basically, just run rmgpy/data/thermoTest.py using your favorite method.

### Expected Behavior
All tests should pass

### Installation Information
Describe your installation method and system information.
 - OS (include version if known): Ubuntu 18
 - Installation method: from source
 - RMG version information:
   <!--
   For binary installation, get version number using `conda list -f rmg`
   For installation from source, get git commit using `git describe --tags`
   -->
   - RMG-Py: master
   - RMG-database: master

--- a/rmgpy/data/thermoTest.py
+++ b/rmgpy/data/thermoTest.py
@@ -54,7 +54,10 @@ def setUpModule():
     """A function that is run ONCE before all unit tests in this module."""
     global database
     database = RMGDatabase()
-    database.load_thermo(os.path.join(settings['database.directory'], 'thermo'))
+    database.load_thermo(
+        os.path.join(settings['database.directory'], 'thermo'),
+        thermo_libraries=['DFT_QCI_thermo', 'SABIC_aromatics', 'primaryThermoLibrary']
+    )
 
 
 def tearDownModule():

